packages/react-ai/src/components/AIConversation/context/SendMessageContext.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SendMessage } from '../../../types';
+
+export const SendMessageContext = React.createContext<SendMessage | undefined>(
+  undefined
```

Comment:
curious why we're using undefined as opposed to null?

Reply:
I don't think there was a particular reason other than to signify that the value has not been set yet. I've noticed we do both `| undefined` or `| null` in our code base so im happy to align them with what we think is correct!

Reply:
Yea I usually see undefined as the default for creating context